style.css

```diff
@@ -28,70 +36,78 @@
     font-family: Arial, Helvetica, sans-serif;
     border: solid 1px #ffe484;
 }
+
 .button-1:hover {
     background: #ffe484;
     color: #563d76;
 }
+
 .main {
     margin: 80px 0px 0px 80px;
+    position: relative;
+    top: 100px;
```

Comment:
Explanation:
Since the `.heading` class is fixed, it covers things after it when the navbar grows taller for smaller screens. We need a way to ensure no matter the situation, the header does not cover things up by default (except when user scroll up).

- Line 47 -> make `.main`, which is the affected `div` that gets covered up by the navbar, have position relative (to the top). Why is it relative to top of page? Since navbar above it is fixed, consider it out of the page and will never listen to margin. So, `.main` is acting as the first `div` now.
- Line 48 -> Create offset from the top by using `top: 100px`. But 100px is not enough to offset a navbar of 160px, nah?!
<img width="496" alt="Screenshot 2019-11-28 at 07 06 24" src="https://user-images.githubusercontent.com/15727660/69781476-93ceb280-11ae-11ea-85d5-dc0eeac43cd9.png">

- Line 46 -> Considering you have `margin: 80px 0px 0px 80px;` which means `margin-top` is 80px, therefore the whole offset is `top: 100px` + `margin-top: 80`, this is `180px` which is 20px longer than `160px`. That ensure `main` is never covered up by `navbar`!

Reply:
Thank you. Is it possible to use a margin-top greater than 160px, or is it better to include the top offset of **position: relative**.